iroh-net/src/derp/client.rs

```diff
@@ -61,7 +61,7 @@ impl Client {
     ///
     /// Errors if the packet is larger than [`super::MAX_PACKET_SIZE`]
     pub async fn send(&self, dstkey: PublicKey, packet: Bytes) -> Result<()> {
-        debug!(%dstkey, len=packet.len(), "[DERP] send");
+        debug!(?dstkey, len = packet.len(), "[DERP] send");
```

Comment:
Could we impl Display again on the keys?  I think before we had these as a short printable version, ~8 hex chars, so that it would be suitable for logging purposes.  It makes logging much easier to read.

Reply:
only keypair is a newtype atm, ao only for that

Reply:
for now both debug and display print the long version